### DISCUSSION TITLE
June 12 Stand Up

### DISCUSSION THREAD

**Discussion by corygalyna** [ZORALab/Hestia#264] [Q&A] [answered]
@hollowaykeanho Update.

**Comment by hollowaykeanho** [comment, marked as answer]
Still porting documentation. Working on `zoralabDLIST` at the moment. 9 components left until that ticket is closed.

**Comment by corygalyna** [comment]
Help?

**Comment by hollowaykeanho** [comment]
https://github.com/ZORALab/Hestia/discussions/259

This takes time.

**Comment by corygalyna** [comment]
Got it.

**Comment by corygalyna** [comment]
That's all?

**Comment by hollowaykeanho** [comment]
Yeah.
